app/static/pages/volunteer/VolunteerProfilesService.js

```diff
@@ -41,23 +41,26 @@ oppiaFoundationWebsite.factory('VolunteerProfilesService', [
     };
     var volunteersByCategory = {};
     Object.keys(VOLUNTEER_INFO).forEach(function(volunteerCategory) {
+
+      console.log(volunteerCategory);
```

Comment:
I don't think you need this logging statement.

Reply:
Deleted!